docs/concepts/consensus/overview.md

```diff
@@ -1,17 +1,17 @@
 # Consensus protocols
 
-GoQuorum implements the following Proof of Authority consensus protocols:
+GoQuorum implements the following proof of authority (PoA) consensus protocols:
 
-* [IBFT](../../configure-and-manage/configure/consensus-protocols/ibft.md)
-* [QBFT](../../configure-and-manage/configure/consensus-protocols/qbft.md)
-* [Raft](../../configure-and-manage/configure/consensus-protocols/raft.md)
-* [Clique](../../configure-and-manage/configure/consensus-protocols/clique.md).
+* [QBFT](../../configure-and-manage/configure/consensus-protocols/qbft.md): QBFT is the recommended enterprise-grade
```

Comment:
Probably don't need to repeat the consensus protocol in these descriptions. For example, we can just say, "QBFT: the recommended enterprise-grade..."